pbt-101.py

```diff
@@ -300,6 +342,13 @@ def test_record_json_roundtrip(record):
     new = Record.from_json(string)
     # TODO: assert that the new and old records match
 
+    # this fails when the Record is 'null',
+    # I don't know how to exclude 'null' from 'Record' in st.builds()
+    # assert new == string
+    # the bigger question is why isn't 'null' equivalent to 'null'?
+    # And isn't 'null' in Python the 'None' type?
+    # Many confuse :-)
```

Comment:
The trick here is that you have to drop a call to `json.loads` in the appropriate place :wink: 

Python's null object is called `None`; and Python is quite strict about comparisons ("strongly typed", meaning no automatic conversions).